### DISCUSSION TITLE
Alert/Notifications for Down Events Only

### DISCUSSION THREAD

**Discussion by rrmrafael** [upptime/upptime#710] [Q&A] [answered]
Hello, currently, alerts or notifications sent are for both down and up events. Is it possible to setup to just send notifications for down events only? Also, can we customize the down event message? I would like to remove the Issues Link.

**Comment by AnandChowdhary** [comment, marked as answer]
Unfortunately there's no way to only allow for down and not up, but you can indeed customize the notification message by adding the secret "NOTIFICATIONS_DOWN_MESSAGE" to your repository: https://github.com/upptime/uptime-monitor/blob/2314e3b25e19e6f4c0736782cabf529ea394b6b0/src/update.ts#L380. For example:

```
$EMOJI $SITE_NAME is $STATUS
```

will replace the emoji with the red/green square, and add the site name and status.
